File: Java/백준/Bronze/25812. Nice Raises/Nice Raises.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int b = in.nextInt();
        int cnt1 = 0;
        int cnt2 = 0;
        for (int i=0;i<a;i++){
            int c = in.nextInt();
            if(b>c){
                cnt1++;
            }
            else if(c>b){
                cnt2++;
            }
        }
        if(cnt1>cnt2){System.out.println(1);}
        else if(cnt2>cnt1){System.out.println(2);}
        else{System.out.println(0);}
    }
}

File: C++11/백준/Bronze/25812. Nice Raises/Nice Raises.cc
#include <iostream>
using namespace std;

int main(){
    int a,b,lim,cnt1=0,cnt2=0;
    cin >> a >> lim;
    for (int i=0;i<a;i++){
        cin >> b;
        if (lim>b)cnt1++;
        else if(b>lim)cnt2++;
    }
    if(cnt1>cnt2){cout<<1;}
    else if(cnt1==cnt2){cout<<0;}
    else{
        cout << 2;
    }
}